src/models/index.js

```diff
@@ -1,32 +1,22 @@
-import Sequelize from 'sequelize';
+import UserModel from './user';
+import MessageModel from './message';
+import mongoose from 'mongoose';
```

Comment:
A think I do in my code which helps to maintain the imported code: Import first the external node packages, then your own source code modules:

```
import mongoose from 'mongoose';

import UserModel from './user';
import MessageModel from './message';
```

Reply:
fixed